### ISSUE TITLE
Make nice Croissant schema identifiers

### ISSUE THREAD

**Issue by david4096** [david4096/huggingface-rdf#1]
They currently have a placeholder base URL when they are not part of an existing schema

**Comment by NuriaQueralt** [comment]
Hi @david4096 @vemonet, should we use https://w3id.org/ as permanent IRI ID? and mint something like `https://w3id.org/biohackathon/CROISSANT_XXXXX` what do you think?

**Comment by david4096** [comment]
It works for me but I don't know the ramifications. Shouldn't MLCommons be doing this?

**Comment by NuriaQueralt** [comment]
Good point! should we contact and ask MLCommons about this?

**Comment by david4096** [comment]
I think Leyla might be able to offer an opinion here. The placeholder is fine for now but I think if the KG became a persistent resource we would want an IRI for the KG as separate from the spec terms. I have my hopes that we can get something like that going but I think for now the best option might be to just allow this to be configurable. Sorry, there are certainly a few unknowns for me in terms of how to approach this.

**Comment by david4096** [comment]
I think we can close this ticket if the CLI feature is added that we can specify the base name as an option.

**Comment by NuriaQueralt** [comment]
Agree. Still for the KG with permanent IRI different from the spec, w3id.org is a good way to go from my experience in my research group and what I see in other initiatives. Maybe @bit2424 from Leyla's lab can shed light on this (I also point Leyla to this issue via Slack)

**Comment by ljgarcia** [comment]
Croissant ML identifiers is within the scope of Croissant ML. Placeholder solution suggested by @david4096 looks fine.
